### PR TITLE
Feature/resync password verif

### PR DESCRIPTION
**Pull request by Firiyuu**
Changes:
1. Added password verification upon resyncing.
2. Error message for wrong password
3. Success message to inform user that resyncing has started.

Issues Found then Fixed:
1. Upon sending ELA, as 1(single digit integer/number) theres a TypError error upon splitting because it only expected values that has a dot or period on it.

### CODE REVIEW

File: src_server/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "fs-extra": "^10.0.0",
     "is-port-reachable": "^3.0.0",
     "morgan": "^1.10.0",
+    "n": "^7.5.0",
```
Comment (bonhokage06) [inline src_server/package.json:17]: this package should not be included in our package.json. Upgrading of nodejs should be included in our system installation.

File: src_server/check_carrier.sh [hunk 2]
```diff
@@ -4,14 +4,10 @@ BOOTSTRAP_FILE=${CARRIER_PATH}/bootstrapd.conf
 CURRENT_IP=$(curl -s ifconfig.me)
 echo "$CURRENT_IP"
 CARRIER_IP=$(grep "external_ip" ${BOOTSTRAP_FILE} | cut -d'"' -f 2)
-echo $CARRIER_IP
-
-if [ "$CURRENT_IP" == "" ]; then
+echo $CARRIER_IPif [ "$CURRENT_IP" == "" ]; then
```
Comment (jhoe123) [inline src_server/check_carrier.sh:7]: @Firiyuu  theres an issue with this file. kindly test and check the console to see and issues

**Review by jhoe123 — COMMENTED** [review]
Theres an issue with check_carrier.sh

**Review by bonhokage06 — APPROVED** [review]

File: src_server/check_carrier.sh (hunk 2 above)
Comment (Firiyuu) [reply src_server/check_carrier.sh]: @jhoe123 I think I messed this up as I tried debugging. I revised it back to original, I didnt meant to change any of check_carrier.sh

**Pull request merged**